src/utils/addProxyIntegrationHeaders.ts

```diff
@@ -2,9 +2,21 @@ import { getProxySecret, WorkerEnv } from '../env'
 
 export function addProxyIntegrationHeaders(headers: Headers, url: string, env: WorkerEnv) {
   const proxySecret = getProxySecret(env)
-  if (proxySecret) {
-    headers.set('FPJS-Proxy-Secret', proxySecret)
-    headers.set('FPJS-Proxy-Client-IP', headers.get('CF-Connecting-IP') || '')
-    headers.set('FPJS-Proxy-Forwarded-Host', new URL(url).hostname)
+  if (!proxySecret) {
+    return
   }
+
+  headers.set('FPJS-Proxy-Secret', proxySecret)
+  headers.set('FPJS-Proxy-Client-IP', getIPFromHeaders(headers))
+  headers.set('FPJS-Proxy-Forwarded-Host', new URL(url).hostname)
+}
+
+export function getIPFromHeaders(headers: Headers) {
+  const connectingIP = headers.get('CF-Connecting-IP')
+
+  if (headers.get('Cf-Pseudo-IPv4') === connectingIP) {
```

Comment:
I would add an explaining comment here

```suggestion
  // Correctly handle Cloudflare's [Pseudo IPv4](https://developers.cloudflare.com/network/pseudo-ipv4/) feature set to `Overwrite headers`
  if (headers.get('Cf-Pseudo-IPv4') === connectingIP) {
```

Reply:
Hey @JuroUhlar thanks for the suggestion. I added the comment with commit [#436e06b6](https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/pull/208/commits/436e06b6eb5b8493521af6053f403c09137d5d0b) and also added you to `Co-Authored-By`. Could you please check and resolve the conversation if it's fixed?